build.sbt

```diff
@@ -15,6 +15,7 @@ lazy val core = project
     addSbtPlugin("de.heikoseeberger"  % "sbt-header"               % "5.6.0"),
     addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.1.0"),
     addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.8.0"),
+    addSbtPlugin("org.scoverage"      % "sbt-scoverage"            % "1.9.3"),
```

Comment:
scoverage used to be problematic in js. Is that still the case?

Reply:
Seems to work now on JS, I tested it in https://github.com/gemini-hlsw/lucuma-sso/pull/380.